### ISSUE TITLE
Add Markdown filter support for Jade

### ISSUE THREAD

**Issue by elisee** [superpowers/superpowers-web#8]
See http://jade-lang.com/reference/filters/

**Comment by mseyne** [comment]
Will this feature replace the markdown slide plugin from https://bitbucket.org/sparklinlabs/superpowers-markslide-system/src ?

I think, to add this func to superpowers-web would be really useful to build website slide/documentation type.

**Comment by elisee** [comment]
Yeah it's part of the work that we want to do to obsolete MarkSlide. It seems unnecessary to have a custom system just for slideshows.

**Comment by icarito** [comment]
Just an idea... what about integrating https://github.com/bevacqua/woofmark ? :-D
